public/src/overmind/actions.tsx

```diff
@@ -478,64 +490,95 @@ export const rebuildSubmission = async ({ state, actions, effects }: Context, {
     if (!(submission && state.selectedAssignment && state.activeCourse)) {
         return
     }
-    const response = await effects.grpcMan.rebuildSubmission(state.selectedAssignment.ID, submission.ID, state.activeCourse)
-    if (success(response)) {
-        // TODO: Alerting is temporary due to the fact that the server no longer returns the updated submission.
-        // TODO: gRPC streaming should be implemented to send the updated submission to the client.
-        await actions.getSubmission({ courseID: state.activeCourse, submission, owner })
-        actions.alert({ color: Color.GREEN, text: 'Submission rebuilt successfully' })
+    const response = await effects.api.client.rebuildSubmissions({
+        courseID: state.activeCourse,
+        assignmentID: state.selectedAssignment.ID,
+        submissionID: submission.ID,
+    })
+    if (response.error) {
+        return
     }
-
+    // TODO: Alerting is temporary due to the fact that the server no longer returns the updated submission.
```

Comment:
Maybe there is an issue for this in the issue tracker, and if not, maybe create one (and reference it in the TODOs here).